frontend/packages/console-shared/src/components/formik-fields/multi-column-field/MultiColumnField.scss

```diff
@@ -18,6 +15,7 @@
     line-height: var(--pf-global--spacer--xl);
     position: absolute;
     right: -30px;
+    top: 0;
```

Comment:
You'll want want to center the button vertically. For a typical row, the result is the same, but gives more flexibility if we start creating rows where the height differs from the button height.
```
top: 50%;
transform: translateY(-50%);
```

This would enable us to support the same layout as is done here:
![image](https://user-images.githubusercontent.com/14068621/80815737-35ae5d00-8b9c-11ea-84ea-c92b9e732ee9.png)

Reply:
Would not just a flex wrapper fix this? There are align center options natively available without going to negative translations.